sources/packages/api/src/services/program-year/program-year.service.ts

```diff
@@ -16,4 +16,20 @@ export class ProgramYearService extends RecordDataModelService<ProgramYear> {
       .orderBy("programYear.id", "DESC")
       .getMany();
   }
+
+  /**
+   * Validate Program Year of an Application,
+   * if the ProgramYear is not active
+   *  return null
+   * else r
+   *  return the ProgramYear row
+   * @param selectedForm Selected Form of the application
+   */
+  async validateProgramYear(selectedForm: string): Promise<ProgramYear> {
```

Comment:
IMO this method is not validating the program year, it is retrieving the data. 
On the controller, it is being used to execute some validation, but this method in the service is not executing a validation, right? Could you please consider rename this method?

Reply:
Is renaming it to selectProgramYear is fine?

Reply:
It seems that the methods retrieving data are using more getSomething, like:
getProgramYearByFormName
getProgramYear
,,,

Reply:
The main reason why I named the method like validateProgramYear is, it actually validates if the programYear is active or not by returning the complete column. But its fine, if you want me to rename